source/advocating.html.erb.md

```diff
@@ -11,41 +11,41 @@ status: editors-draft
 [ ... Tip description goes here ... ]
 
 {:.attach_permalink}
-## Develop accessibility business case
+## Develop an accessibility business case
 
 [ ... Tip description goes here ... ]
 
 {:.attach_permalink}
-## Secure management support for accessibility activities
+## Address management to support accessibility activities
```

Comment:
I think the structure of the statement in the original makes a bit more sense. Unless there is some different meaning that you are looking to include?